flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -1869,4 +1908,16 @@ public boolean isForce() {
 			return props.forceCheckpoint();
 		}
 	}
+
+	private enum CoordinatorRestore {
+
+		/** Coordinators are always restored. If there is no checkpoint, they are restored empty. */
+		ALWAYS,
```

Comment:
nit: the following set of names would be more clear to me: `SKIP, RESTORE_IF_PRESENT, RESTORE_OR_RESET`

Reply:
I'll change this to `RESTORE_OR_RESET`, `RESTORE_IF_CHECKPOINT_PRESENT`, `SKIP`.